spring-cloud-azure-autoconfigure/src/main/java/com/microsoft/azure/spring/cloud/autoconfigure/telemetry/PropertyLoader.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+package com.microsoft.azure.spring.cloud.autoconfigure.telemetry;
+
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+import lombok.extern.java.Log;
+import org.springframework.lang.NonNull;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+@Log
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class PropertyLoader {
+
+    private static final String PROJECT_PROPERTY_FILE = "/META-INF/project.properties";
+    private static final String APPLICATION_PROPERTY_FILE = "/application.properties";
```

Comment:
I'm unsure whether introducing another property file just for access to project version is a good idea. I find this, but don't know whether it works. https://stackoverflow.com/questions/2712970/get-maven-artifact-version-at-runtime